tests/test_new_excerpt.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import time
+import requests
+from new_excerpt import new_excerpt
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+@pytest.mark.parametrize("file_name, file_format", [("gdb",'id_formats_1'), ("shp",'id_formats_2'), 
```

Comment:
Please use 2 newlines to separate standalone function (the ones that are not inside of classes). (Same is true for classes)